common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -29,33 +29,68 @@
  * according to a specific format.
  */
 public class FormatUtils {
+  /**
+   * Parse a list of <code>Objects</code> into a <code>String</code>.
+   *
+   * @param objs a list of Objects to convert to a String
+   * @return comma-separated concatenation of the string representation returned by Object#toString
+   *         of the individual objects
+   */
   public static String parametersToString(Object... objs) {
     StringBuilder sb = new StringBuilder("(");
-    for (int k = 0; k < objs.length; k ++) {
-      if (k != 0) {
-        sb.append(", ");
+    if (objs != null) {
+      for (Object obj : objs) {
+        if (sb.length() != 1) {
```

Comment:
Can obj.toString() ever return an empty string? Actually, probably, if the actual emtpy string is a parameter.

If so, then this condition would behave differently from the previous code. What should happen if toString() returns an empty string? There should still be an entry for it?

And, can you add a test case for this case?